test/karma.conf.js

```diff
@@ -74,7 +73,7 @@ module.exports = (config) => {
     browserNoActivityTimeout: 300000,
     captureTimeout: 180000,
 
-    browsers: ['ChromeHeadlessLocal'],
+    browsers: ['ChromeHeadless'],
```

Comment:
Actually the real fix is to fix the browser name in the `karma-browsers.json`. I messed up the search and replace and didn't really realize that it also affected `ChromeHeadlessLocal`.

Reply:
Fixed.